hardhat.config.ts

```diff
@@ -0,0 +1,39 @@
+import * as dotenv from "dotenv";
+
+import { HardhatUserConfig } from "hardhat/config";
```

Comment:
I had trouble importing the `hardhat.config` from the `contracts-v2/dist` folder as it couldn't import the files imported by that file. So I copied it directly into here and removed all nonessential parts